pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -91,6 +91,7 @@ type BackupStorageConfig struct {
 	AzureStorageContainer string                `json:"azureStorageContainer,omitempty"`
 	AzureResourceGroup    string                `json:"azureResourceGroup,omitempty"`
 	GcpBucket             string                `json:"gcpBucket,omitempty"`
+	AwsCustomCABundle     []byte                `json:"awsCustomCABundle,omitempty"`
```

Comment:
I know this name is just following the prior convention for S3 related stuff `Aws[..]`, but still feels strange to me since the only time we would fill this value out would be if our S3 store was _not provided by AWS_. 

Anyone have thoughts on `S3CustomCABundle` for this param name instead?

Reply:
> Also, nit, but thoughts on moving this param up to be grouped with other AWS / S3 params?

Yeah, I'll move that up with the others.

Reply:
Yeah, I'll move that up with the others.

Reply:
The _custom_ in the name seems redundant.  Isn't this a custom bundle by definition?  Could it be anything else?  Seems `S3CABundler` would have been sufficient.  Along the same lines, is there any reason to only scope it to S3?

Reply:
I borrowed the naming convention from the aws-sdk, it's `CustomCABundle` there, but just `CABundle` would probably be better. If we anticipate this being used in other storage backends, then there's no reason to scope it to S3.

Reply:
"there's no reason to scope it to S3" -- hmm, on this point, it seems that the convention we're following is that everything is scoped to the provider.